### PR TITLE
loading .xml config file containing the data sources using Jackson xml

### PR DESCRIPTION
**Pull request by SrdjanStevanetic**
Hi Sergiu,

please review the implementation of the ticket EA-2357. Thank you!

Best Regards, Srdjan

### CODE REVIEW

File: entity-management-web/pom.xml [hunk 1]
```diff
@@ -107,6 +108,23 @@
...
+        
+        <!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml -->
+		<dependency>
+		    <groupId>com.fasterxml.jackson.dataformat</groupId>
+		    <artifactId>jackson-dataformat-xml</artifactId>
+		    <!-- version>2.11.4</version-->
+		</dependency>
+		
+		<!-- https://mvnrepository.com/artifact/junit/junit -->
+		<dependency>
```
Comment (gsergiu) [inline entity-management-web/pom.xml:120]: we have to use junit jupitter, see user sets tests for appropriate configurations

File: entity-management-web/src/test/java/eu/europeana/entitymanagement/config/DataSourcesDeserializationTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ * JUnit test to check if DataSources are properly deserialized from XML
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
```
Comment (gsergiu) [inline entity-management-web/src/test/java/eu/europeana/entitymanagement/config/DataSourcesDeserializationTest.java]: change to junit jupitter test

File: pom.xml [hunk 3]
```diff
@@ -26,6 +40,8 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
 
+        <springCore.version>4.3.23.RELEASE</springCore.version>
```
Comment (gsergiu) [inline pom.xml]: please use the same spring core version as in API Commons

**Review by gsergiu — COMMENTED** [review]
please address review comments

File: entity-management-web/pom.xml (hunk 1 above)
Comment (SrdjanStevanetic) [reply entity-management-web/pom.xml:120]: Changed!

File: entity-management-web/src/test/java/eu/europeana/entitymanagement/config/DataSourcesDeserializationTest.java (hunk 2 above)
Comment (SrdjanStevanetic) [reply entity-management-web/src/test/java/eu/europeana/entitymanagement/config/DataSourcesDeserializationTest.java]: Changed!

File: pom.xml (hunk 3 above)
Comment (SrdjanStevanetic) [reply pom.xml]: Updated.

**Pull request merged**